wolfcrypt/src/evp.c

```diff
@@ -2593,6 +2595,40 @@ int wolfSSL_EVP_PKEY_CTX_set_signature_md(WOLFSSL_EVP_PKEY_CTX *ctx,
     return WOLFSSL_SUCCESS;
 }
 
+int wolfSSL_EVP_PKEY_CTX_set_rsa_pss_saltlen(WOLFSSL_EVP_PKEY_CTX *ctx,
```

Comment:
Consider wrapping this with `#ifndef WC_RSA_PSS`. Even consider rework for `saltlen` in `struct WOLFSSL_EVP_PKEY_CTX`